### PR TITLE
Homework19.2

### PR DESCRIPTION
**Pull request by Hronologiya**

### CODE REVIEW

File: lesson_19_2/test_app.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import os
+import requests
+import tempfile
+
+def test_upload_retrieve_image():
```
Comment (DartBrovsky) [inline lesson_19_2/test_app.py]: It is not the first time I highlight that it is necessary to create test cases inside Test Class. That is best practice of Test Automation approach. Please fix that in your code.

File: lesson_19_2/test_app.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    # Create a temporary directory
+    with tempfile.TemporaryDirectory() as temp_dir:
+        # Create a temporary image file
+        temp_file_path = os.path.join(temp_dir, 'image.jpg')
+        with open(temp_file_path, 'wb') as temp_file:
+            # Upload the image
+            response = requests.post('http://127.0.0.1:8080/upload', files={'image': open(temp_file_path, 'rb')})
+            assert response.status_code == 201
```
Comment (DartBrovsky) [inline lesson_19_2/test_app.py]: It is better to use 3rd party libraries for assertions. For ex. assertpy. Also it is always better to add failed state descriptions  for your test cases assertions.

File: lesson_19_2/test_app.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            image_url = response.json()['image_url']
+
+            # Retrieve the image
+            response = requests.get(f'http://127.0.0.1:8080/image/{image_url.split("/")[-1]}', headers={'Accept': 'image/jpeg'})
```
Comment (DartBrovsky) [inline lesson_19_2/test_app.py]: get method should be inside separate test case. It is not so good to mixed up all tests together. Especially when your task was to write 3 test cases - for post, get and delete.

**Review by DartBrovsky — CHANGES_REQUESTED** [review]
Hi, I reviewed and leave several comments to fix. Thanks.

File: lesson_19_2/test_app.py (hunk 1 above)
Comment (Hronologiya) [reply lesson_19_2/test_app.py]: Added class

File: lesson_19_2/test_app.py (hunk 2 above)
Comment (Hronologiya) [reply lesson_19_2/test_app.py]: Add assertpy

File: lesson_19_2/test_app.py (hunk 3 above)
Comment (Hronologiya) [reply lesson_19_2/test_app.py]: Make changes.

**Review by DartBrovsky — APPROVED** [review]

**Pull request merged**